test/models/admin_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdminTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
don't leave comment code without any explanation. Either remove the code, or put comment why you commented it